### PR TITLE
Several improvements; please see the commit messages

### PR DESCRIPTION
**Pull request by ADTC**
Please see the individual commits for more details. Thank you. [You can see the preview here.](https://github.com/ADTC/nlp-resume-parser)

Note: All the steps in the _Readme_ are numbered with `1.` to make maintenance easier. You can add/remove steps or re-arrange them without having to fix the numbering. Markdown to HTML engine will automatically number it correctly.

### CODE REVIEW

File: application/logs/.gitkeep
Comment (hxu296) [inline application/logs/.gitkeep:1]: Please keep the .gitkeep or Github will delete the `logs` directory, as Github only tracks files not directories. Other changes look good :)

**Review by hxu296 — CHANGES_REQUESTED** [review]
Thank you for your contribution👏 nit: please keep the `application/logs/` directory. I believe the logger will fail otherwise, There is unfortunately no handle to automatically create a log directory if it doesn't exist yet. Other changes look great!

PS: thank you for the md number trick!

File: application/logs/.gitkeep
Comment (ADTC) [reply application/logs/.gitkeep:1]: > There is unfortunately no handle yet to automatically create a log directory if it doesn't exist.

@hxu296 Couldn't we just add a mkdir command to the run script? We do that for the uploads directory.

Comment (ADTC) [reply application/logs/.gitkeep:1]: Build script I mean.

Comment (ADTC) [reply application/logs/.gitkeep:1]: It's similar to PR #2. I think that will do. I will test it out in a bit.

Comment (ADTC) [reply application/logs/.gitkeep:1]: Hi Huan Xu @hxu296

I've tested it on a clean copy of the repo. It works fine to just create the logs directory as part of the build script. There's no need for a `.gitkeep` file.

I've added the command in commit 658967b (just pushed it, see below). The `-p` option ignores the mkdir attempt when the directory already exists.

Thank you for your review and kind words btw. I hope this is sufficient for a merge. Thanks in advance for the merge. 🙂 

PS: Let's avoid using "squash & merge" as I'd like to keep the changes logically separated in commits.

Comment (hxu296) [reply application/logs/.gitkeep:1]: Looks great! Thank you for testing on a clean copy :)

**Pull request merged**